circeSupport/src/test/scala/CirceSupportSpec.scala

```diff
@@ -0,0 +1,48 @@
+import io.buildo.enumero._
+import io.buildo.enumero.circe._
+import io.circe._, io.circe.syntax._, io.circe.parser._
+
+import org.scalatest.{Matchers, WordSpec}
+
+class CirceSupportSpec extends WordSpec with Matchers {
+  sealed trait Planet extends CaseEnum
+  object Planet {
+    case object Mercury extends Planet
+    case object Venus extends Planet
+    case object Earth extends Planet
+  }
+
+  val planetMap = Map[Planet, Int](
+    Planet.Mercury -> 12,
+    Planet.Venus -> 812763,
+    Planet.Earth -> 0
+  )
+
+  val json = planetMap.asJson
```

Comment:
I'd move this into the test, serialisation should break if `CirceSupport` is not working as intended.